format_checker/forked-projects.json

```diff
@@ -523,4 +524,4 @@
   "https://github.com/zeroturnaround/gradle-jrebel-plugin": "unforked",
   "https://github.com/zhangxd1989/spring-boot-cloud": "unforked",
   "https://github.com/zhurlik/gradle-jboss-modules-plugin": "unforked"
-}
```

Comment:
Please don't make these unnecessary changes. It doesn't matter if they were made automatically by your editor/IDE. Just configure it to not make such changes.